api/team.go

```diff
@@ -723,6 +730,28 @@ func getTeamByName(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getMyTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
+	if result := <-Srv.Store.Team().GetTeamsForUser(c.Session.UserId); result.Err != nil {
+		c.Err = result.Err
+		return
+	} else {
+		data := result.Data.([]*model.TeamMember)
+		w.Write([]byte(model.TeamMembersToJson(data)))
+	}
+}
+
+func getMyTeamsUnread(c *Context, w http.ResponseWriter, r *http.Request) {
+	teamId := r.URL.Query().Get("id")
+
+	if result := <-Srv.Store.Team().GetTeamsUnreadForUser(teamId, c.Session.UserId); result.Err != nil {
```

Comment:
We should think about caching or etag, if this is going to be called all the time.